src/stories/Library/pause-reservation/PauseReservation.tsx

```diff
@@ -0,0 +1,32 @@
+import ToggleButton from "../toggle-button/ToggleButton";
+
+export type PauseReservationProps = {
+  isChecked: boolean;
+};
+
+export const PauseReservation = ({ isChecked }: PauseReservationProps) => {
+  return (
+    <div className="dpl-pause-reservation-component">
+      <div className="dpl-pause-reservation-component__pagefold" />
+      <div className="dpl-pause-reservation-component__flex">
+        <div className="dpl-pause-reservation-component__flex__reservation-icon">
+          <img src="icons/collection/Reservations.svg" alt="Reservations" />
+        </div>
+        <div className="dpl-pause-reservation-component__flex__text">
+          Sæt fysiske reserveringer på pause
+        </div>
+        {isChecked && (
+          <span
+            aria-label="Reservationer er sat på pause i følgende periode"
+            className="dpl-pause-reservation-component__flex__badge"
+          >
+            20.09.2021 - 12.10.2021
+          </span>
+        )}
+        <div className="dpl-pause-reservation-component__flex__button">
+          <ToggleButton isChecked={isChecked} />
+        </div>
+      </div>
+    </div>
+  );
+};
```

Comment:
Shouldn't this file be under Blocks, since it is showing a composition which includes the toggle button?

Reply:
I don't think so, as I dont think this is a standalone component, but rather something to nest in another component